components/eam/src/physics/crm/sam/crm_module.F90

```diff
@@ -81,14 +81,21 @@ subroutine crm(lchnk, ncrms, dt_gl, plev,       &
     !-----------------------------------------------------------------------------------------------
     ! Interface variable declarations
     !-----------------------------------------------------------------------------------------------
-
-    integer , intent(in   ) :: lchnk                            ! chunk identifier (only for lat/lon and random seed)
     integer , intent(in   ) :: ncrms                            ! Number of "vector" GCM columns to push down into CRM for SIMD vectorization / more threading
     integer , intent(in   ) :: plev                             ! number of levels in parent model
     real(r8), intent(in   ) :: dt_gl                            ! global model's time step
     type(crm_input_type), target,intent(in   ) :: crm_input
     type(crm_state_type), target,intent(inout) :: crm_state
-    type(crm_rad_type),   target,intent(inout) :: crm_rad
+    real(crm_rknd), dimension(ncrms,crm_nx_rad,crm_ny_rad,crm_nz), intent(inout) :: crm_rad_qrad
```

Comment:
Why did we need to undo the encapsulation of crm_rad?

Reply:
as discussed on slack - I've reverted the interface changes but kept a few changes in crm_module to make the 2-mom sections consistent with the 1-mom sections.